Dfe.Data.Common.Infrastructure.CognitiveSearch/Filtering/SearchFilterRequest.cs

```diff
@@ -50,4 +55,17 @@ public SearchFilterRequest(string filterKey, IEnumerable<object> filterValues)
         FilterKey = filterKey;
         FilterValues = filterValues.ToArray();
     }
+
+    /// <summary>
+    /// Allows a filter value delimiter to be specified, if required by the underlying expression type.
+    /// </summary>
+    /// <param name="filterValuesDelimiter">
+    /// The string used to delimit filter values.
+    /// </param>
+    public void SetFilterValuesDelimiter(string filterValuesDelimiter)
+    {
+        ArgumentNullException.ThrowIfNullOrWhiteSpace(filterValuesDelimiter);
```

Comment:
so the delimiter isn't allowed to be a space?

Reply:
Change made to ArgumentException.ThrowIfNullOrEmpty(filterKey);